drivers/iommu/intel/iommu.c

```diff
@@ -63,6 +63,8 @@
 			    (pdev)->device == 0x4e19 ||		\
 			    (pdev)->device == 0x465d ||		\
 			    (pdev)->device == 0x1919))
+#define IS_IPTS(pdev) ((pdev)->vendor == PCI_VENDOR_ID_INTEL &&	\
+			    ((pdev)->device == 0x9d3e))
```

Comment:
Do we need to add more IDs here? CC @StollD

Reply:
We should probably at least add `34E4`, for Icelake devices, assuming they have the same issue. There are other IDs, but these are not found on surfaces (https://github.com/torvalds/linux/blob/master/drivers/misc/mei/hw-me-regs.h, marked with `iTouch`)

Reply:
Do we have any indication that those devices are affected as well?

Reply:
I dont remember any report about it on gen7, but then, reports for anything touch related are sparse on gen7.

Reply:
Okay. I've tried to search for IOMMU issues on gen7 but haven't found anything... so I'd wait until we have confirmation that it's needed.